.gitignore

```diff
@@ -44,6 +44,66 @@ mathenv
 cover/
 .konchrc
 
+# Node
+######
+
+# Logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
```

Comment:
cannot recall off hand, does `node_modules/` match nested sub directories? asking because I've also seen `**/node_modules` before but maybe that was just another way of saying the same thing?

Example: admin has its own package.json -> `/admin/node_modules`